CRM/Upgrade/Incremental/sql/5.0.2.mysql.tpl

```diff
@@ -0,0 +1,4 @@
+{* file to handle db changes in 5.0.2 during upgrade *}
+
+ALTER TABLE civicrm_case_type
+ADD COLUMN category VARCHAR(512);
```

Comment:
This one I added myself since I checked the "value" column in the option value table and it's VARCHAR(512). The DAO was generated automatically, not sure why it decided this should be an int.

Even if the reference column (value) is VARCHAR(512), after investigating for a bit I'll change the category column to INT(10) because that's what core uses:

Upgrader: https://github.com/compucorp/civicrm-core/blob/8f9a245ba678dcfe2a023f85060d065989c1926d/CRM/Upgrade/Incremental/php/FourFour.php#L597
Schema: https://github.com/compucorp/civicrm-core/blob/8f9a245ba678dcfe2a023f85060d065989c1926d/xml/schema/Activity/ActivityContact.xml#L58
Dao: https://github.com/compucorp/civicrm-core/blob/8f9a245ba678dcfe2a023f85060d065989c1926d/CRM/Activity/DAO/ActivityContact.php#L57

Thanks for pointing this out (though I really don't know what's going on in core's head)